rust/actyx/ax/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2021"
 name = "ax"
-version = "0.1.0"
+version = "2.17.0"
```

Comment:
please add a build.rs check that this string matches `ax_core::DATABANK_VERSION`